tests/integration/test_analysisd/test_eps/test_invalid_values.py

```diff
@@ -0,0 +1,78 @@
+import os
+import pytest
+
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.modules.eps import event_monitor as evm
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.modules.eps import ANALYSISD_STATE_INTERNAL_DEFAULT
+from wazuh_testing.processes import check_if_deamon_is_not_running
+
+
+pytestmark = [pytest.mark.server]
```

Comment:
Add tier to the test

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3181/commits/f817116435639fa720611bd68eb90ced7c606062)